outbox/README.md

```diff
@@ -10,7 +10,7 @@ excluding any duplicate messages by comparing incoming event ids with already su
 
 Prepare the Java components:
 
-    mvn clean install
+    mvn clean install -Pnative -Dnative-image.docker-build=true
```

Comment:
So I reckon this requires Graal to be installed locally, right? How about pushing the build into a (multi-stage) Docker build, simplifying that set-up: https://github.com/debezium/debezium-examples/blob/master/kstreams/poc-ddd-aggregates/Dockerfile